instance/models/mixins/openedx_theme.py

```diff
@@ -65,15 +67,13 @@ def get_theme_settings(self):
             return ""
 
         # These settings set the values required by simple_theme
-        settings = {
+        theme_settings = {
             # This block defines our theme by applying the chosen colors to SASS-defined color variables
             "SIMPLETHEME_SASS_OVERRIDES": [
                 {
                     "variable": "link-color",
                     "value": application.link_color,
                 },
-                # TODO: These are specific to Ginkgo and can be removed
```

Comment:
@giovannicimolin, you can remove the enclosed styles as per the removed comment. :smiley: